apps/client/src/services/p2p-transport.ts

```
@@ -35,41 +39,14 @@ export class P2PTransport {
-
-    logger.info(`Initiating connection with ${clientId}`);
+  initiate(clientId: ClientId): void {
+    logger.info(`Initiating with ${clientId}`);
 
     this.ensurePeer(clientId, true);
   }
 
-  async initiateAll(clientIds: ClientId[]): Promise<void> {
-    await Promise.all(clientIds.map((id) => this.initiate(id)));
+  initiateAll(clientIds: ClientId[]): void {
+    clientIds.forEach((id) => this.initiate(id));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)